src/main/java/seedu/address/logic/parser/EditDeckCommandParser.java

```diff
@@ -30,7 +29,7 @@ public EditDeckCommand parse(String args) throws ParseException {
             index = ParserUtil.parseIndex(i);
```

Comment:
We definitely need to find a different way to parse string.  `args.charAt(1)` will not work if the index has 2 digits eg. 11, because the index parsed will become 1 and not 11!